src/test/java/marquez/db/JobRunDaoTest.java

```diff
@@ -82,6 +80,9 @@ public static void setup() throws MarquezServiceException {
     CREATED_JOB_RUN_UUID = createdJob.getNamespaceGuid();
   }
 
+  @BeforeClass
+  public static void setup() throws MarquezServiceException {}
```

Comment:
Why introduce an empty class-level setup method?

Reply:
This method shouldn't have been introduced. Was something I was experimenting with the get a test to pass.